app/models/detector/citation.rb

```diff
@@ -69,6 +72,12 @@ def initialize(phrase)
       @score = calculate_score
     end
 
+    def detections
+      return unless detection?
```

Comment:
I'm not (yet) convinced the use case you are suggesting is worth having this work differently than the other models which only return detections that they are treating as meaningful. It feels a bit odd to return an array of `detections` when we don't have any `detections?` though.

I believe I understand the use case for development and will give it a bit more thought and would welcome additional perspective you may want to add.

Reply:
I've been thinking about this more since my initial review, and I'm not sure that this module needs to change to accommodate the use case I'm thinking of. The citation detector's structure is unique enough in this regard that any more complete reporting method might just need to be added to that one detector, at least until there's other tools that are similar.

Reply:
I think one option for the situation you are describing in that you want to see things in this method that wouldn't normally return in development as the score is too low to be considered a match would be to set the `REQUIRED_SCORE` to 0 during your development run. Doing so will return all data for all Terms.

Reply:
Additional commit addresses adds a note to `REQUIRED_SCORE` that it can be set lower to see more in development if desired. I'm not sure anyone will think to look there but I couldn't think of a better place that someone would look and wasn't weird.